### DISCUSSION TITLE
Multiple wokers in a server

### DISCUSSION THREAD

**Discussion by hwwxj** [giongto35/cloud-game#251] [Q&A] [answered]
exm, can we create multiple workers in a server?  and how can make it?

**Comment by sergystepanov** [comment, marked as answer]
Of course, you can, that was one of the benefits of such master/slave architecture where a number of workers controlled through a coordinator app could fully utilize available server resources / run multiple games on one server.
Essentially, just run an instance of the coordinator app, multiple instances of workers, and point them to the coordinator app.
As a dumb example for the native run (better use a proper Bash script):
```
./bin/coordinator --v=5 &
./bin/worker --coordinatorhost localhost:8000 &
./bin/worker --coordinatorhost localhost:8000 &
./bin/worker --coordinatorhost localhost:8000 &
...
```
With the current docker-compose run it can be similar in here:
```
command: >
      bash -c "Xvfb :99 & coordinator --v=5 & worker --coordinatorhost localhost:8000"
```
And true Docker containerization approach is not tested yet, with the coordinator container and several worker containers. But, I think, it should work with some hacking.

**Reply by hwwxj** [reply]
Thanks!
